bundles/org.eclipse.swt.svg/src/org/eclipse/swt/svg/JSVGRasterizer.java

```diff
@@ -0,0 +1,160 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Vector Informatik GmbH and others.
```

Comment:
It's 2024 already (here and below) :) And even 2025 soon 😨
```suggestion
 * Copyright (c) 2024 Vector Informatik GmbH and others.
```

Reply:
Thanks for the hint. :) Fixed in 1ccfe6580fff2145e1c59987649dddb8b1fe0ed8